workers/unity/Assets/Generated/Source/improbable/gdk/tests/nonblittabletypes/NonBlittableComponentReaderWriter.cs

```diff
@@ -19,7 +19,7 @@ public partial class Requirables
             [InjectableId(InjectableType.ReaderWriter, 1002)]
             internal class ReaderWriterCreator : IInjectableCreator
             {
-                public IInjectable CreateInjectable(Entity entity, EntityManager entityManager, ILogDispatcher logDispatcher)
+                public IInjectable CreateInjectable(Entity entity, EntityManager entityManager, World world, ILogDispatcher logDispatcher)
```

Comment:
nit: if we pass in the world, we don't need to pass in the entity manager. 
or we could also just pass in the worker as it has a reference to the logdispatcher and the world

Reply:
Agreed, will adjust here

Reply:
Chose to pass the world, passing the worker would require a bigger change

Reply:
I can't find a case where World is used in this function. Which class uses it?